### DISCUSSION TITLE
Projects (beta) Access for Collaborators

### DISCUSSION THREAD

**Discussion by markdorison** [community/community#4906] [Projects and Issues] [answered]
I don't see any mention in the [Projects (beta) documentation](https://docs.github.com/en/issues/trying-out-the-new-projects-experience) on permissions. Who can see new (beta) Projects and who can not? Is it all org members? I don't see a way to add collaborators like I can with (non-beta) Projects. Is it safe to assume this is something that would roll out later?

**Comment by galah92** [comment]
This is very much required for my organization tool

**Comment by stephaniefois** [comment]
There is [documentation](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/managing-access-to-projects#granting-a-collaborator-access-to-your-project) for this but following these steps does not actually allow for me to add a collaborator. Searching the name of a collaborator does not bring up any results to then invite.

**Comment by azenMatt** [comment]
Hi 👋 - we are working on adding outside collaborators at the moment - if you would like to be part of the beta for this please let me know your org name and I can add you to the test

**Reply by stephaniefois** [reply]
yes please! My organization is called Viget Labs.

**Reply by azenMatt** [reply]
Hi @stephaniefois - `vigetlabs` has been added 🎉 

As a heads up - this list of possible outside collaborators is anyone who is already a collaborator on a repo. Let me know how it goes!

**Reply by HGZdev** [reply]
Hello @azenMatt  
We would be glad to test this functionality as well. My organization: [StratoKit](https://github.com/StratoKit)

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - thanks for you help in testing this, it is now live! more details in our latest changelog - https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/
